packages/react/src/components/Footer/Footer.js

```diff
@@ -6,26 +6,75 @@
  */
 
 import React from 'react';
-import { localeSelector } from '../../internal/FeatureFlags';
-import LocaleSelector from './LocaleSelector';
-
-/** Footer component */
-class Footer extends React.Component {
-  /**
-   * Renders the footer component
-   *
-   * @returns {object} JSX object
-   */
-  render() {
-    return (
-      <footer>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-        {localeSelector ? (
-          <LocaleSelector text="Load the Locale Selector!" />
-        ) : null}
-      </footer>
-    );
+import PropTypes from 'prop-types';
+// import { localeSelector } from '../../internal/FeatureFlags';
+// import LocaleSelector from './LocaleSelector';
+import { settings } from 'carbon-components';
```

Comment:
@photodow this import should point to the es folder so we don't pull in the whole package. I think this is what's causing our artifact to balloon up to be huge. I already asked @kennylam to do the same for the Masthead.
cc: @annawen1

Reply:
like...

```javascript
import { settings } from 'carbon-components/es';
```

or are you asking for this...?

```javascript
import settings from 'carbon-components/es/globals/js/settings';
```

Reply:
@photodow the latter. @kennylam already tried this on his end and it drastically reduced the file size.